hack/make/.ensure-syscall-test

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+# Build a C binary for cloning a userns for seccomp tests
+# and compile it for target daemon
+(
+	if [ "$(go env GOOS)" = "linux" ]; then
```

Comment:
Shouldn't this line be using the env variable for the Docker Engine `GOOS` (`DOCKER_ENGINE_GOOS`)? (sorry, didn't notice this one before)